source/_docs/configuration/yaml.markdown

```diff
@@ -111,3 +111,16 @@ This means that you've mistakenly entered a tab character, instead of spaces.
 Home Assistant is case sensitive, a state of `'on'` is not the same as `'On'` or `'ON'`. Similarly an entity of `group.Doors` is not the same as `group.doors`.
 
 If you're having trouble, check the case that Home Assistant is reporting in the dev-state menu, under *Developer tools*.
+
+### {% linkable_title Can't find configuration.yaml %}
```

Comment:
This should be part of the troubleshooting section because it's only relevant for Hass.io users.